### PR TITLE
debian/control: Fix syntax error causing builds to fail

### PR DESCRIPTION
**Pull request by qwertychouskie**
whoops my bad

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -19,7 +19,7 @@ Description: An open source re-implementation of RollerCoaster Tycoon 2.
  The gameplay revolves around building and maintaining an amusement park
  containing attractions, shops and facilities. The player must try to make a
  profit and maintain a good park reputation whilst keeping the guests happy.
- 
+ .
```
Comment (janisozaur) [inline debian/control:22]: Well is this 
image
 all over again?

**Review by janisozaur — APPROVED** [review]
https://www.debian.org/doc/debian-policy/ch-controlfields.html#id24

File: debian/control (hunk 1 above)
Comment (qwertychouskie) [reply debian/control:22]: Luckily the Debian package description is not connecting to sandboxing working or not 😄

Comment (janisozaur) [reply debian/control:22]: So we're told ;)

**Pull request merged**